Commands/Process.php

```diff
@@ -37,13 +35,15 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $systemCheck = new SystemCheck();
         $systemCheck->checkRedisIsInstalled();
 
-        Access::getInstance()->setSuperUserAccess(false);
-        Plugin\Manager::getInstance()->setTrackerPluginsNotToLoad(array('Provider'));
+        $trackerEnvironment = new Environment('tracker');
```

Comment:
it doesn't recreate the container, it creates a new one w/ the tracker environment. For now, this pushes a container instance onto a stack so StaticContainer::get() will use the top. Which I think means I missed something, there should probably be a `$trackerEnvironment->destroy()` call after processing is done, so control goes back to the CLI environment.

Note: This means there's a new Plugin\Manager and should be new Plugin instances stored in `$trackerEnvironment`. Since it's only done once here, shouldn't be an issue.

Eventually, we'll get rid of StaticContainer so everything is provided through DI, then we just use the environment directly and worry about nothing.